server/cluster/cluster.go

```diff
@@ -536,6 +538,14 @@ func (c *Cluster) allocTableID(ctx context.Context) (uint64, error) {
 	return id, nil
 }
 
+func (c *Cluster) allocShardID(ctx context.Context) (uint32, error) {
+	id, err := c.shardIDAlloc.Alloc(ctx)
+	if err != nil {
+		return 0, errors.Wrap(err, "cluster alloc shard id failed")
```

Comment:
```suggestion
		return 0, errors.WithMessage(err, "cluster alloc shard id failed")
```

Reply:
The error style has been updated by #55.

Reply:
fix

Reply:
fix